v4/source/system.yaml

```diff
@@ -646,3 +646,90 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  '/svrbusy':
+    post:
+      tags:
+        - system
+      summary: Set the server busy (high load) flag
+      description: |
+        Marks the server as currently having high load which disables non-critical services such as search, statuses and typing notifications.
+        ##### Permissions
+        Must have `manage_system` permission.
+      parameters:
+        - name: secs
+          in: query
+          required: false
+          description: Number of seconds until server is automatically marked as not busy.
+          default: "3600"
+          type: string
+      responses:
+        '200':
+          description: Server busy flag set successfully
+          schema:
+            $ref: "#/definitions/StatusOK"
+        '400':
+          $ref: '#/responses/BadRequest'
+        '403':
+          $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
```

Comment:
Can you add the `lang: 'curl'` option here?

Reply:
curl sample added to all three endpoints via https://github.com/mattermost/mattermost-api-reference/commit/e26aa85745a0df9b21b3b642d3f7efeb1b05de95 

curl samples tested as well.